### PR TITLE
SDK-27: add support for TC output file types

### PR DESCRIPTION
**Pull request by mayamessinger**

### CODE REVIEW

**Comment by mayamessinger** [comment]
Tests pass
test

File: lib/rev-api/models/order_request.rb [hunk 1]
```diff
@@ -101,24 +101,42 @@ def validate_inputs(inputs)
...
 
     # Optional, should we include timestamps?
     attr_reader :timestamps
 
+    # All supported output file formats
+    OUTPUT_FILE_FORMATS = {
+      :ms_word => 'MS Word',
+      :json => 'JSON',
+      :text => 'Text',
+      :pdf => 'Pdf'
```
Comment (mgray2010) [inline lib/rev-api/models/order_request.rb:120]: Do these work with Timi's recent changes?

**Comment by mgray2010** [comment]
As far as my extensive years of ruby development go, this LGTM after integration testing :shipit:

File: lib/rev-api/models/order_request.rb (hunk 1 above)
Comment (mayamessinger) [reply lib/rev-api/models/order_request.rb:120]: Yes, it looks like it https://github.com/revdotcom/revdotcom/pull/19036/

Timi's changes made parsing strings case-insensitive so case is fine, and the `output_file_formats` param is still the same name

**Comment by mayamessinger** [comment]
Integration test (against api sandbox) worked - default is MS word, but my request specified JSON and JSON was returned

proof

**Pull request merged**